### PR TITLE
SRCH-4529 Create the localization YAML file for Haitian Creole

### PR DESCRIPTION
**Pull request by SamiraMSadat**
## Summary
- Add Haitian Creole localization file
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: config/locales/ht.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+# It's safe to modify the order of these (e.g., '%m/%d/%Y'), but do not localize the Latin letters 'd', 'm', or 'Y'. Do not change the case of the letters.
+  cdr_format: '%d/%m/%Y'
+  before: 'Anvan'
+  clear: 'Efase'
+
+# Prompts for spelling, spelling overrides, and did you mean suggestions
+  showing_results_for: Ցուցադրվում է որոնման արդյունքներ %{corrected_query}
+  search_instead_for: Փոխարենը որոնել %{original_query}
+
+# Prompts for searchers to revise or refine search
```
Comment (dawnpm) [inline config/locales/ht.yml:42]: @SamiraMSadat did you get ht localizations for lines 42-55?

File: config/locales/ht.yml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  last_year: "Ane pase"
+  custom_range: 'Peryòd pèsonalize'
+  from: 'Soti'
+  to: 'Rive'
+# It's safe to modify the order of these (e.g., '%m/%d/%Y'), but do not localize the Latin letters 'd', 'm', or 'Y'. Do not change the case of the letters.
+  cdr_format: '%d/%m/%Y'
+  before: 'Anvan'
+  clear: 'Efase'
+
+# Prompts for spelling, spelling overrides, and did you mean suggestions
```
Comment (dawnpm) [inline config/locales/ht.yml:38]: @SamiraMSadat did you get ht localizations for lines 38-55?

**Review by dawnpm — COMMENTED** [review]
@SamiraMSadat a question about missing changes. I'll check with a dev about the failed CI run.

**Comment by SamiraMSadat** [comment]
@dawnpm Checking on this! I implemented the requested changes and merged in main. Let me now if there is anything else needed on my end.

File: config/locales/ht.yml [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      search_again: "Refè rechèch la ankò"
+# Prompts for searchers to view more results from the broader website after initially seeing site-scoped results
+    site_limits:
+      including_results_for_query_from_matching_sites: "N ap enkli rezilta yo pou %{query} ki soti nan %{matching_sites} sèlman."
+      do_you_want_to_see_results_for: "Èske w vle wè rezilta yo pou %{query_from_all_sites}?"
+      query_from_all_sites: '%{query} ki soti tout kote'
+# Label for inline news module and prompt to see more news on news vertical
+    more_news_about_query: 'Plis %{news_label} konsènan %{query}'
+    news_about_query: '%{news_label} konsènan %{query}'
+    news_search_options: "Sijè ki an rapò yo"
```
Comment (dawnpm) [inline config/locales/ht.yml:84]: @SamiraMSadat it turns out we do not want a localization on this line. Please remove, so it just says `news_search_options:`

File: config/locales/ht.yml [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+# It's safe to modify the order of these (e.g., 'm/d/yyyy'), but do not localize 'd', 'm', or 'yyyy'. Do not change the case of the letters.
+    date_format: 'm/d/yyyy'
```
Comment (dawnpm) [inline config/locales/ht.yml:86]: please indent this line so it's two space characters in from the `news_search_options:` item just above.

**Review by dawnpm — DISMISSED** [review]
Hi @SamiraMSadat , we finally had a chance to figure out what was going on. please see inline comments below. Thanks!

**Comment by SamiraMSadat** [comment]
Thanks @dawnpm ! I just addressed your comments, looks like it passes now

**Review by stevenbarragan — APPROVED** [review]
LGTM

**Pull request merged**